src/components/pages/books/books.css

```diff
@@ -185,3 +185,41 @@
 .addBtn:hover {
   cursor: pointer;
 }
+
+@media screen and (max-width: 768px) {
+  .book-section-one {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
```

Comment:
| Zeplin's Design |
|-|
| ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/9b797c9f-7439-4d7f-81ef-35f324a369fd) |

| @AleWaweru Design|
|-|
| ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/133dc289-5435-4ff3-9d0e-b3a2a7b491e3) |

- [ ] It would be nice if you follow the Zeplin's design because your color for the buttons is not very similar to the given design :fearful: Kindly make them the same color as Zeplin's design to make your project follows the project requirements :+1: 
- [ ] It would be nice if you make your Circular progress bar the same color as the Zeplin's design, this means the color of the progress bar is not the same and then it would be great if you add the percentage as well which is missing in your design :fearful: Kindly make these changes to make your design attractive :+1:
- [ ] It would be nice if you follow the Zeplin's design and add the Chapter number to the top of the Update Progress button to make your design look similar to the Zeplins design :+1: